proto/vtctldata.proto

```diff
@@ -1230,6 +1231,17 @@ message UpdateCellsAliasResponse {
   topodata.CellsAlias cells_alias = 2;
 }
 
+message WorkflowUpdateRequest {
```

Comment:
(⚠️ extremely-annoying, i know) nit: i've been trying, Sisyphean-like, to keep these request/responses (and the RPC listing in `vtctlservice.VtctldServer`, so this is a dual-purpose comment) sorted for easier human scanning